lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -47,14 +47,35 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     super._descendantRemoved.apply(this, arguments);
   }
 
+  _getElementNodes() {
+    return domSymbolTree.treeToArray(this.getRootNode({}), {
+      filter: node => {
+        if (!this._filterElement(node)) {
+          return false;
+        }
+
+        const owner = formOwner(node);
+        if (owner && this === owner) {
+          return true;
+        }
+
+        return false;
+      }
+    });
+  }
+
+  _filterElement(node) {
```

Comment:
Since this function is used in only one place now, it would be simpler to inline it into the call site.